docs/reference/errors-and-warnings/NU1901-NU1904.md

```diff
@@ -51,3 +51,7 @@ For example `<NuGetAuditSuppress Include="https://github.com/advisories/GHSA-g3q
 
 If you do not want NuGet to check for packages with known vulnerabilities during restore, add `<NuGetAudit>false</NuGetAudit>` inside a `<PropertyGroup>` in your project file, or a [`Directory.Build.props` file](/visualstudio/msbuild/customize-by-directory).
 If you would like to run NuGet Audit on developer machines, but disable it on CI pipelines, you can take advantage of MSBuild importing environment variables, and create a NuGetAudit environment variable set to `false` in your pipeline definition.
+
+In NuGet 6.12 (Visual Studio/MSBuild 17.12 and .NET 9.0.100 SDK), NuGet changed the default for `NuGetAuditMode` to `all`, which means report on transitive packages with known vulnerabilities.
+The value can be explicitly set to `direct` to revert back to .NET 8's default.
+Alternatively, [the property `SdkAnalysisLevel`](/dotnet/core/project-sdk/msbuild-props#sdkanalysislevel) can be set to `8.0.400` to lock multiple NuGet and .NET SDK features to .NET 8.0.400 defaults.
```

Comment:
Should we call out the scenario which is, if you prefer not to address it now, you can temporarily set etc. 

The spirit of SDK Analysis level is until, behave like an older SDK until I have time to address it.